action.yml

```diff
@@ -1,16 +1,22 @@
-name: 'Close milestones that are at 100%'
-description: 'Automatically closes milestones that have no more open issues/PRs ✨'
-author: 'Instantish'
+name: "Close finished milestones - by Instantish"
+description: "Automatically closes milestones that have no more open issues/PRs ✨"
+author: "Instantish"
 branding:
-  icon: 'book'  
-  color: 'green'
+  icon: "check"
+  color: "gray-dark"
 inputs:
   repo-token:
-    description: 'Token for the repository. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`.'
+    description: "Token for the repository. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`."
     required: true
+  minimum-issues:
+    description: "Check milestones that contains the minimum issues count, otherwise ignore it moving to the next."
+    default: "3"
+  related-only:
```

Comment:
Seems this option is only for debugging, and not sure we need it, since in non-debug mode, we set this based on the type of event triggered the action.

Reply:
you meant the `related-only`? with that the user could separate if he wants that the pr itself collect the linked milestone, or the PR would collect all issues/pr milestones (but yes you have a point here, we could force the user to optimize triggers on actions set) and if we go by that, then we just set that the pr auto triggers the milestone it links to, instead of every milestone then we remove this option.